hermes/src/api/types.rs

```diff
@@ -19,23 +19,21 @@ use {
         Price,
         PriceIdentifier,
     },
-    utoipa::{
-        openapi::{
-            RefOr,
-            Schema,
-        },
-        IntoParams,
-        ToSchema,
-    },
+    utoipa::ToSchema,
     wormhole_sdk::Chain,
 };
 
 
-/// PriceIdInput is a wrapper around a 32-byte hex string.
-/// that supports a flexible deserialization from a hex string.
-/// It supports both 0x-prefixed and non-prefixed hex strings,
-/// and also supports both lower and upper case characters.
-#[derive(Debug, Clone, Deref, DerefMut)]
+/// A price id is a 32-byte hex string, optionally prefixed with "0x".
+/// Price ids are case insensitive.
+///
+/// Examples:
+/// * 0x63f341689d98a12ef60a5cff1d7f85c70a9e17bf1575f0e7c0b2512d48b1c8b3
+/// * 63f341689d98a12ef60a5cff1d7f85c70a9e17bf1575f0e7c0b2512d48b1c8b3
+///
+/// See https://pyth.network/developers/price-feed-ids for a list of all price feed ids.
+#[derive(Debug, Clone, Deref, DerefMut, ToSchema)]
+#[schema(value_type=String)]
```

Comment:
I tried for a while, but couldn't figure out how to get utoipa to enforce that this matches a regex for hexadecimal strings 🤷

Reply:
I think it's fine